pkg/scaffold/v1/crd/crd_sample.go

```diff
@@ -28,6 +28,7 @@ import (
 var _ input.File = &CRDSample{}
 
 // CRDSample scaffolds a manifest for CRD sample.
+// nolint:golint
```

Comment:
It is because of the name `crd.CRD`. In this case, we can move for the suggestion as well which could be crd.Sample{} instead of. After think better, shows make more sense. I will change it as well.

Reply:
I would change it to keep it consistent with the change to Prometheus, so that the change is project-wide.

I would agree with you about the deprecation of `v1` if it wasn't a 10 secs fix.

Reply:
In order to keep it "project-wide." I am not in favor to do this change since the name in the v2 for the same manifest is == equals and will NOT face the same because it's implementation is not in the crd pkg.

Reply:
Oh, I see what you mean.